pkg/cli/scaffold.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cli
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+// NewCommand return a new scaffold command
+func (c *CLI) newScaffoldCmd() *cobra.Command {
```

Comment:
This feels like it should be in the `alpha` folder but since I believe I need methods from `CLI` I cannot use a different package.

Reply:
@camilamacedo86 looking for some guidance here. Since the `scaffold` command is going to be essionaially doing an `init` + `edit` + `create`, I need access to the`cli` package.